test/test_transforms_v2_consistency.py

```diff
@@ -1,256 +1,30 @@
-    except Exception as exc:
-        raise AssertionError(
-            "Initializing the prototype transform failed with the error above. "
-            "This means there is a consistency bug in the constructor."
-        ) from exc
-
-    check_call_consistency(
-        prototype_transform,
-        legacy_transform,
-        images=make_images(**config.make_images_kwargs),
-        supports_pil=config.supports_pil,
-        closeness_kwargs=config.closeness_kwargs,
-    )
-
-
-@pytest.mark.parametrize(
-    ("config", "args_kwargs"),
-    [
-        pytest.param(
-            config, args_kwargs, id=f"{config.legacy_cls.__name__}-{idx:0{len(str(len(config.args_kwargs)))}d}"
-        )
-        for config in CONSISTENCY_CONFIGS
-        for idx, args_kwargs in enumerate(config.args_kwargs)
-        if not isinstance(args_kwargs, NotScriptableArgsKwargs)
-    ],
-)
-def test_jit_consistency(config, args_kwargs):
-    args, kwargs = args_kwargs
-
-    prototype_transform_eager = config.prototype_cls(*args, **kwargs)
-    legacy_transform_eager = config.legacy_cls(*args, **kwargs)
-
-    legacy_transform_scripted = torch.jit.script(legacy_transform_eager)
-    prototype_transform_scripted = torch.jit.script(prototype_transform_eager)
-
-    for image in make_images(**config.make_images_kwargs):
-        image = image.as_subclass(torch.Tensor)
-
-        torch.manual_seed(0)
-        output_legacy_scripted = legacy_transform_scripted(image)
-
-        torch.manual_seed(0)
-        output_prototype_scripted = prototype_transform_scripted(image)
-
-        assert_close(output_prototype_scripted, output_legacy_scripted, **config.closeness_kwargs)
-
-
-class TestToTensorTransforms:
-    def test_pil_to_tensor(self):
-        prototype_transform = v2_transforms.PILToTensor()
-        legacy_transform = legacy_transforms.PILToTensor()
-
-        for image in make_images(extra_dims=[()]):
-            image_pil = to_pil_image(image)
-
-            assert_equal(prototype_transform(image_pil), legacy_transform(image_pil))
-
-    def test_to_tensor(self):
-        with pytest.warns(UserWarning, match=re.escape("The transform `ToTensor()` is deprecated")):
-            prototype_transform = v2_transforms.ToTensor()
-        legacy_transform = legacy_transforms.ToTensor()
+from torchvision.transforms import functional as legacy_F
+from torchvision.transforms.v2 import functional as prototype_F
+from torchvision.transforms.v2._utils import _get_fill, query_size
+from torchvision.transforms.v2.functional import to_pil_image
 
-        for image in make_images(extra_dims=[()]):
-            image_pil = to_pil_image(image)
-            image_numpy = np.array(image_pil)
 
-            assert_equal(prototype_transform(image_pil), legacy_transform(image_pil))
-            assert_equal(prototype_transform(image_numpy), legacy_transform(image_numpy))
+def make_bounding_boxes(*args, batch_dims, **kwargs):
```

Comment:
This was the only change I needed to make to be able to drop the whole `transforms_v2_legacy_utils` module. I'm ok with that trade, since we either won't touch this file or might kill them as well (see https://github.com/pytorch/vision/pull/8013#issue-1922629262).

Reply:
Ok, I reverted this in 8d04cb27e89313ccbfd51d0bd2bcfc596b30122f since I forgot about the prototype tests. We need to port them of the legacy utils as well at some point, since these are not really legacy tests. But this for another PR.